src/style.css

```diff
@@ -0,0 +1,158 @@
+html {
+  width: 100%;
+  height: 100%;
+}
+
+* {
+  margin: 0 !important;
+  padding: 0 !important;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+main {
+  width: 480px !important;
+  padding: 3px !important;
+  border-radius: 0.2em;
+  box-shadow: 0 0 6px rgb(43, 42, 42);
+  padding-top: 1em;
+  padding-bottom: 1em;
+}
+
+header {
+  width: 100% !important;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 3px;
+  background-color: white;
+  border-top-right-radius: 0.2em;
+  border-top-left-radius: 0.2em;
+}
+
+form {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom-style: solid;
+  border-bottom-color: rgb(148, 147, 147);
+  border-bottom-width: 1px;
+  margin-bottom: 3px;
+  height: 35px;
+  background-color: white;
+  padding-left: 12px !important;
+  padding-right: 12px !important;
+}
+
+input.addTaskInput,
+.addTaskButton {
+  height: 25px;
+  font-size: 20px;
+  border: none !important;
+  box-shadow: none !important;
+  border-radius: 0;
+  -webkit-appearance: none;
+  margin: 0;
+  background-color: white;
+}
+
+input {
+  border: none;
+  background: transparent;
+  width: 90%;
+}
```

Comment:
-[ ] it is better to put the general rule of inputs as the first rule before start the specific rules of each input